datasm/tools/consolidated_cmip_dataset_report.py

```diff
@@ -705,6 +706,7 @@ def main():
     skipcount = 0
 
     for dsid_key in esgf_report:
+        print(f"DEBUG: stage 4: dsid_key = {dsid_key}", file=sys.stderr, flush=True)
```

Comment:
Should we use `log_message()` to replace this print statement?

Reply:
Actually, this "tool" is independent of datasm, imports no datasm-specific modules.  I simply placed it under the tools directory to put it under git control.  I guess it should be in a separate git repo.  (For that matter, so should the e2c cwl workflows...).

Reply:
Gotcha